### DISCUSSION TITLE
- (`__sub__`) operator for histograms

### DISCUSSION THREAD

**Discussion by beojan** [scikit-hep/boost-histogram#538] [Ideas]
Histograms can already be subtracted by adding -1 times the RHS. It would therefore be nice to have a `-` operator on histograms.
Personally I find this useful when comparing histograms (I tend to prefer plotting `(h2 - h1) / h1` instead of `h2/h1` since the sign lets you see the direction of movement easily).

**Comment by henryiii** [comment]
@HDembinski What do you think?
